### PR TITLE
Add realpath to testoval.py

### PR DESCRIPTION
**Pull request by redhatrises**
- This should fix the occasional cannot find content error message

### CODE REVIEW

File: shared/utils/testoval.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 import os
 
 # always use shared/testoval_module.py version
-script_directory = os.path.dirname(__file__)
+script_directory = os.path.dirname(os.path.realpath(__file__))
 SHARED_MODULE_PATH = os.path.join(script_directory, "..", "modules")
+print script_directory
```
Comment (mpreisler) [inline shared/utils/testoval.py]: yeah... no..

Comment (redhatrises) [reply shared/utils/testoval.py]: @mpreisler lol. yeah totally didn't mean to commit that.

Comment (redhatrises) [reply shared/utils/testoval.py]: @mpreisler just making sure that you were watching what I was doing. ;)
It should be fixed now.

Comment (mpreisler) [reply shared/utils/testoval.py]: I see everything, Gah-bay! :D

Comment (redhatrises) [reply shared/utils/testoval.py]: @mpreisler lol are you sauron the all seeing eye? :P

**Pull request merged**